### ISSUE TITLE
Somebody show me how to run this project? 

### ISSUE THREAD

**Issue by imax16041998** [yfgeek/BlockVotes#23]
I have problem with command `php composer.phar install`: 
"Could not open input file: composer.phar"
Please contact my email: **imax16041998@gmail.com**

**Comment by imax16041998** [comment]
When i clicked "Send An Email" then nothing happen. Why?
![image](https://user-images.githubusercontent.com/50973410/101361101-24b39000-38d1-11eb-9ad3-43ee1492f253.png)

**Comment by metalfiiish** [comment]
Have you tried turning on Web developer tools to trace the web calls in the "Network" tab with 'preserve log' checked? As well, enabling verbose level in the "Console" tab can show any script related errors. 

That may point you to any blocked port traffic or script errors.

**Comment by imax16041998** [comment]
@metalfiiish Do you have run this project successfully?
Can you give me your email to contact?

**Comment by hafizahkassim99** [comment]
@imax16041998 can you run this Project ?

**Comment by chess254** [comment]
Yes, i run it once sometime back, needs lots of reconfiguration thoughOn Apr 23, 2021 9:26 PM, hafizahkassim99 ***@***.***> wrote:
@imax16041998 can you run this Project ?

**Comment by hafizahkassim99** [comment]
@chess254 can you pls contact me ... this is my e-mail : nurhafizahkassim@gmail.com 
I act can run this Project ... its just tht I stuck at the tallying page... would really appriciate if you could help me 🥺
